index.html

```diff
@@ -54,5 +57,174 @@ <h1>I'm Fernando Glad to see <br> you!</h1>
 				</ul>
 			</div>
 		</section>
+
+		<section class="works">
+			<div class="card">
+				<img src="img/img1.jpg" alt="" />
+				<h1>Tonic</h1>
+				<div class="card-details">
+					<h3>canopy</h3>
+					<i class="fas fa-circle"></i>
+					<h4>back end dev</h4>
+					<i class="fas fa-circle"></i>
+					<h4>2015</h4>
+				</div>
+				<p class="card-description">
+					A daily selection of privately personalized reads; no accounts or
+					sign-ups required.
+				</p>
+				<div class="card-skills">
+					<ul>
+						<li><a href="#">html</a></li>
+						<li><a href="#">css</a></li>
+						<li><a href="#">javaScript</a></li>
+					</ul>
+				</div>
+				<button class="btn">see project</button>
+			</div>
+
+			<div class="card">
+				<img src="img/img2.jpg" alt="" />
+				<h1>Multi-Post Stories</h1>
+				<div class="card-details">
+					<h3>canopy</h3>
+					<i class="fas fa-circle"></i>
+					<h4>back end dev</h4>
+					<i class="fas fa-circle"></i>
+					<h4>2015</h4>
+				</div>
+				<p class="card-description">
+					A daily selection of privately personalized reads; no accounts or
+					sign-ups required.
+				</p>
+				<div class="card-skills">
+					<ul>
+						<li><a href="#">html</a></li>
+						<li><a href="#">css</a></li>
+						<li><a href="#">javaScript</a></li>
+					</ul>
+				</div>
+				<button class="btn">see project</button>
+			</div>
+
+			<div class="card">
+				<img src="img/img3.jpg" alt="" />
+				<h1>Tonic</h1>
+				<div class="card-details">
+					<h3>canopy</h3>
+					<i class="fas fa-circle"></i>
+					<h4>back end dev</h4>
+					<i class="fas fa-circle"></i>
+					<h4>2015</h4>
+				</div>
+				<p class="card-description">
+					A daily selection of privately personalized reads; no accounts or
+					sign-ups required.
+				</p>
+				<div class="card-skills">
+					<ul>
+						<li><a href="#">html</a></li>
+						<li><a href="#">css</a></li>
+						<li><a href="#">javaScript</a></li>
+					</ul>
```

Comment:
- [ ] I'm so impressed with how you used the correct tags `ul` and `li` here 🔥. However, please remove the `a` tag from the lists as the list here is just a list of technologies and not a link 😄.